src/widget.js

```diff
@@ -246,7 +246,7 @@ Widget.prototype = {
     // Initial button
     this.rsInitial.addEventListener('click', () => {
       // choose backend only if some providers are declared
-      if (this.rs.apiKeys) {
+      if (this.rs.apiKeys && Object.keys(this.rs.apiKeys).length > 0) {
```

Comment:
I don't entirely understand this one. If `apiKeys` is an empty object by default, we wouldn't need the first check, no?